src/ConfigFactory.Avalonia.Demo/Models/DemoConfig.cs

```diff
@@ -78,4 +81,35 @@ public static ObservableCollection<string> GetThings()
             return value is true;
         });
     }
+
+    /// <summary>
+    /// Example implementation of i18n using the .resx Embedded Resource
```

Comment:
Might just be me, but I had to google `i18n`. Better to use the full description for clarity.

```suggestion
    /// Example implementation of internationalization and localization using the .resx Embedded Resource
```

Reply:
Noted.